lib/ClangImporter/ImportDecl.cpp

```diff
@@ -158,32 +159,26 @@ createVarWithPattern(ASTContext &ctx, DeclContext *dc, Identifier name, Type ty,
 static FuncDecl *createFuncOrAccessor(ASTContext &ctx, SourceLoc funcLoc,
                                       Optional<AccessorInfo> accessorInfo,
                                       DeclName name, SourceLoc nameLoc,
-                                      ParameterList *bodyParams,
-                                      Type resultTy,
-                                      bool throws,
-                                      DeclContext *dc,
+                                      GenericParamList *genericParams,
+                                      ParameterList *bodyParams, Type resultTy,
+                                      bool throws, DeclContext *dc,
```

Comment:
I completely understand what you're saying. These parameters got reformatted because I added `genericParams`. But, I'll remove the header-re-arranging (or put it in another PR).